hw/ip/otbn/util/Makefile

```diff
@@ -2,26 +2,52 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-pylibs := insn_yaml.py mem_layout.py
-pyscripts := yaml_to_doc.py otbn-as otbn-ld otbn-objdump
-
 .PHONY: all
 all: lint asm-sanity
 
-lint-targets := $(addprefix lint-,$(pyscripts))
-.PHONY: lint $(lint-targets)
-lint: $(lint-targets)
+# We need a directory to build stuff and use the "otbn/util" namespace
+# in the top-level build directory.
+build-dir := ../../../../build/otbn/util/code-snippets
```

Comment:
I'd suggest `build-bin` rather than `build` as `build` is where fusesoc places things but `build-bin` is where the software binaries end up. There's also `build-out` which effectively the scratch directory for the meson build system where intermediate files as well as final binaries get placed (which are then copied to `build-bin`). I used `build-bin` for the smoke test as it seemed more appropriate (plus it's called `$BIN_DIR` in `utils/build_consts.sh` where `build-out` is `$OBJ_DIR`).

Reply:
Though I'm not too fussed about `build-bin` vs `build-out`. When we have a 'proper' OTBN software build setup we'll no doubt need to have this conversation again but for now I think we can just choose one and ensure both these tests and smoke use the same place.

Reply:
Cool, sounds sensible. I've moved it to `build-bin` for now.